packages/geoview-core/src/core/components/geolocator/geolocator.tsx

```diff
@@ -122,7 +135,7 @@ export function Geolocator() {
   };
 
   return (
-    <Box sx={sxClasses.root} id="geolocator-search">
+    <Box sx={sxClasses.root} visibility={`${active ? 'visible' : 'hidden'}`} id="geolocator-search">
```

Comment:
Can you try use this code for this line? I don't think we need `$` :
`<Box sx={sxClasses.root} visibility={active ? 'visible' : 'hidden'} id="geolocator-search" />`

Reply:
Mostly in react they handle `if..else` conditions like this.. `{ active & <Box .../>}`.. if this is if..else for show/hide.